src/OpenDebugAD7/Telemetry/DebuggerTelemetry.cs

```diff
@@ -210,6 +213,7 @@ public void SendTelemetryEvent(string eventName, Dictionary<string, object> even
             properties[TelemetryEngineVersion] = _engineVersion;
             properties[TelemetryHostVersion] = _hostVersion;
             properties[TelemetryAdapterId] = _adapterId;
+            properties[TelemetryFrameworkVersion] = _frameworkVersion;
```

Comment:
>properties[TelemetryFrameworkVersion] = _frameworkVersion; [](http://example.com/codeflow?start=12&length=58)

We might want to be a bit more choosy on what events we add this property -- seems unnecessary to have this on all events.